From 68932296f01cd3eee3ee47f3e8f339f33c18b95d Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Tue, 19 May 2015 19:47:52 +0200
Subject: [PATCH] units: make sure systemd-nspawn@.slice instances are actually
 located in machine.slice

https://plus.google.com/112206451048767236518/posts/SYAueyXHeEX
(cherry picked from commit 45d383a3b888195b01b58dbd2c46a11027ff5022)

Cherry-picked from: 45d383a
Resolves: #1222517
---
 units/systemd-nspawn@.service.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/units/systemd-nspawn@.service.in b/units/systemd-nspawn@.service.in
index 3e26b53..5e86121 100644
--- a/units/systemd-nspawn@.service.in
+++ b/units/systemd-nspawn@.service.in
@@ -17,6 +17,7 @@ KillMode=mixed
 Type=notify
 RestartForceExitStatus=133
 SuccessExitStatus=133
+Slice=machine.slice
 Delegate=yes
 
 [Install]
